### ISSUE TITLE
trezor: Model T and Model One not detected

### ISSUE THREAD

**Issue by teknico** [decred/decrediton#3978]
Operating system: `Linux Debian testing`, up to date on Dec. 3 2024, kernel `6.12.1-1-liquorix-amd64`
Decrediton: v. `2.0.5`
Model T firmware: v. `2.8.1` released Aug. 21 2024
Model One firmware: v. `1.12.1` released Mar. 15 2023

udev rules installed, both device files created as `/dev/trezor1` and `/dev/trezor2` resp.

Both devices correctly recognized by the Trezor [web app](https://suite.trezor.io/web), which updated both firmwares without issues.

Clicking on "Setup a Trezor Wallet in Decrediton" shows this message:

![Screenshot_20241203_111734](https://github.com/user-attachments/assets/e87c2303-0cb8-4723-aa47-4ce8ea6b98a7)

The Trezor bridge, however, is [deprecated](https://trezor.io/learn/a/what-is-trezor-bridge). Nothing happens when clicking on the "Connect to Trezor" button.

**Comment by JoeGruffins** [comment]
I tried with `2.8.1` and it is working on linux in the dev enviroment. Will check the release...

**Comment by JoeGruffins** [comment]
It seems the release binary has not been built with the trezor bridge included. It does not work for me either. I also didn't know the bridge was deprecated, so wonder what we are supposed to do there from now. Will look into it.

**Comment by teknico** [comment]
I built the [Trezor bridge daemon](https://github.com/trezor/trezord-go) with Go 1.23.6 (it [doesn't build](https://github.com/trezor/trezord-go/issues/303) with the just released 1.24.0). After running it, Decrediton finds the Model T and creates a Trezor-based wallet. However:

- it shows the wallet as a read-only one;
- upon opening it, the Staking tab is missing;
- the Receive DCR tab shows the warning below.

It looks like more work  is needed in Decrediton to enable the staking feature.

![Image](https://github.com/user-attachments/assets/d046de1b-d0f0-4ae0-9e36-5335cc1bb8bf)

**Comment by teknico** [comment]
Indeed, the needed work is being tracked in #2681 , with some of it already done in #3848 . @JoeGruffins , I know you’re busy with dex work, but finishing this would be great. Thanks. 🙏🏼

**Comment by JoeGruffins** [comment]
@teknico Looks like the daemon is included in the latest release, can you confirm it works on 2.0.6? https://github.com/decred/decred-binaries/releases

If it looks ok please close this issue.

The missing stake section and warning were included with ledger integration and is correct currently.

Waiting for review on #3848 before continuing with trezor ticket purchases.

**Comment by teknico** [comment]
@JoeGruffins Decrediton does indeed connect to the Trezor device, without having to start the bridge daemon manually.

However, after canceling the creation of a Trezor-based wallet, Decrediton loses the connection to the wallet and is unable to access the wallet again until Decrediton is restarted.

Who could help with reviewing #3848 ?
